tests/server.py

```diff
@@ -41,3 +45,49 @@ def test_generic_host_product_name(fixture):
     dmi = parse(fixture)
     server = ServerBase(dmi)
     assert server.get_product_name() == 'SR'
+
+@parametrize_with_fixtures(
+    'dmidecode/', only_filenames=[
+        'HP_SL4540_Gen8',
+    ])
+def test_create_server(fixture):
+    # FIXME: use docker image fixtures
+    try:
+        dc = nb.dcim.sites.create(
+            name='DC1',
+            slug='dc1',
+        )
+    except:
+        dc = nb.dcim.sites.get(slug='dc1')
```

Comment:
Your test should be deterministic so you shouldn't need to do any try-except, I guess?